website/docs/providers/type/community-index.html.markdown

```diff
@@ -17,39 +17,44 @@ please fill out this [community providers form](https://docs.google.com/forms/d/
 
 <table class="table">
 
+    <td><a href="https://github.com/abiquo/terraform-provider-abiquo">Abiquo</a></td>
```

Comment:
Unrelated to the change here, it appears the leading `<tr>` is missing before this line?

Reply:
oh interesting, i'll add it in the next PR

Reply:
added!